### DISCUSSION TITLE
How to set temperature when using the python module?

### DISCUSSION THREAD

**Discussion by kalolad1** [simonw/llm#255] [Q&A]
I saw on the docs you can set the temperature option through the CLI. Is there a way to do this in python?

**Comment by ccwork288** [comment]
See, https://llm.datasette.io/en/stable/python-api.html.

```
model = llm.get_model("gpt-3.5-turbo")
model.key = "... key here ..."
print(model.prompt("Names for otters", temperature=0.2))
```

**Comment by theresia** [comment]
The doc has included an example of how to specify options for the model to use with the -o parameter https://llm.datasette.io/en/stable/usage.html#executing-a-prompt
